src/parser/core/CombatLogParser.js

```diff
@@ -175,20 +165,21 @@ import StormglideSteps from '../shared/modules/items/bfa/raids/crucibleofstorms/
 import TridentOfDeepOcean from '../shared/modules/items/bfa/raids/crucibleofstorms/TridentOfDeepOcean';
 import VoidStone from '../shared/modules/items/bfa/raids/crucibleofstorms/VoidStone';
 import ZaxasjsDeepstriders from '../shared/modules/items/bfa/raids/crucibleofstorms/ZaxasjsDeepstriders';
-// Azsharas Eternal Palace
```

Comment:
poor Eternal Palace

Reply:
Yeah I am killing off one of them - we have 2x TEP, next commits cleans it up a bit more.